conf/modules.config

```diff
@@ -632,11 +633,13 @@ if (!params.skip_consensus_peaks) {
 
         withName: 'SUBREAD_FEATURECOUNTS'  {
             ext.args   = '-F SAF -O --fracOverlap 0.2'
+            ext.prefix = { "${meta.id}.consensus_peaks" }
```

Comment:
Have gone so many times through this file... Much better with your corrections 😄